18 ReactHotLoader/package.json

```diff
@@ -15,19 +16,26 @@
     "lodash": "^4.5.1",
     "object-assign": "^4.0.1",
     "q": "^1.4.1",
-    "react": "^15.2.1",
+    "react": "^15.3.0",
     "react-dom": "^15.2.1",
     "react-redux": "^4.4.5",
     "react-router": "^2.5.2",
     "redux": "^3.5.2",
     "redux-thunk": "^2.1.0",
-    "toastr": "^2.1.2"
+    "toastr": "^2.1.2",
+    "webpack-dev-server": "^1.14.1"
   },
   "devDependencies": {
+    "babel-core": "^6.13.2",
```

Comment:
After testing to run app without babel-loader see [this comment](https://github.com/Lemoncode/react-typescript-samples/pull/69/files#diff-3ad904bbc923f4b79d2d513654a0026cR43), I think we can remove all dependencies with babel